pom.xml

```diff
@@ -42,7 +42,8 @@
   </modules>
 
   <properties>
-    <compiler.version>1.8</compiler.version>
+    <maven.compiler.source>8</maven.compiler.source>
```

Comment:
Does it actually support 8 interchangeably with 1.8? I can find it immediately in the docs and all examples seem to use 1.8

Reply:
Ok, it's just [-source](https://github.com/apache/maven-compiler-plugin/blob/b225b1c29b1ee2fc2b178d2f2f1b920afe1d9a75/src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java#L180) compiler option and from the javac help:

```
  --source <release>, -source <release>
        Provide source compatibility with the specified Java SE release. Supported releases: 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18
```
so it should be good.

Reply:
Those properties will be automatically picked up by other plugins  (for example the [maven-javadoc-plugin](https://maven.apache.org/plugins/maven-javadoc-plugin/jar-mojo.html#source)) while the `compiler.version` is not...and yes you can use `8` instead of `1.8` (related to the docs that looks like a lack of detailed docs..)...also so the ran builds show that...